### ISSUE TITLE
[A11y][WCAG2.1][ARIA]Some status/alert messages are not being read out.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8284]
Concerning the following:
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1061326
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1061322
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1069948

Fixes  need to be made for the following pages/scenarios:
1. Copy for install instructions on package details page
Steps to Reproduce :
    1. Launch application using provided URL.(https://www.nuget.org/)
    2. Navigated to Home page
    3. Search for a key term (say “json”)
    4. Look at the package search results and select any package.
    5. Navigate till "Package Manager" and select it.
    6. Start Narrator and navigate till "Copy" control.
    7. Hit enter on the "Copy" control and observe the announcement by screen reader.
2. API Key Generation success message
Steps to Reproduce :
    1. Launch application using provided URL.(https://www.nuget.org/)
    2. Navigated to Home page
    3. Click on “Sign in” option (top right) and click on “Sign in with Microsoft” button
    4. Sign in using MSA on the common Microsoft sign-in page
    5. Go to “API Keys” from the context menu (top left menu dropdown).
    6. Navigate till "Create Control" and expand it.
    7. Fill all the data in the editing fields.
    8. Start Narrator, click on the "Create" button and observe the announcement by screen reader.
3. Deprecation dialog status messages
Repro Steps:​ Note Same issue was observed on "GitHub Usage" page
    1. Start Screen Reader. Hit the URL "https://www.nuget.org" and sign in with valid credentials​
    2. Tab till "Sign In" link in the top navigation and press enter​
    3. Tab till "Sign in with Microsoft" button and press enter​
    4. Select valid user id and press enter​
    5. Enter any Username, tab till "Register" button and press enter​
    6. Tab till the user menu button and press enter to expand it. Select "Manage Packages" and press enter​
    7. Tab till "Published Packages", expand it and select any existing package.(If no packages are available, add a new package)​
    8. Tab till "Manage Package" link under "Info" and press enter​
    9. Tab till "Deprecation" section and press enter to expand it​
    10. Tab till "Select reason(s)" and uncheck all the check boxes. A status message will be displayed on the screen
    11. Check if the screen reader is reading the status message​

Notes:
All of these are likely missing aria-live regions describing them. Including an aria-live and role to the respective elements that need to be read out should be sufficient. See https://github.com/NuGet/NuGetGallery/pull/8238 for an example of a fix.
